src/nxdoc/nuxeo-add-ons/amazon-s3-online-storage.md

```diff
@@ -250,6 +244,10 @@ nuxeo.s3storage.crypt.key.alias=the_key_alias
 nuxeo.s3storage.crypt.key.password=the_key_password
 ```
 
+{{#> callout type='note' }}
```

Comment:
Should this be at warning level?
what's happen if if you don't do that?
it it creates a situation where existing blobs cannot be read and new one are differently encrypted, resulting in a corrupted instance it should be a warn

Reply:
>it creates a situation where existing blobs cannot be read and new one are differently encrypted and result is a corrupted bucket 

Exact, let's say it deserves a `warn` indeed